graylog2-web-interface/src/components/datanode/hooks/useShowDatanodeMigration.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+import { useQuery } from '@tanstack/react-query';
+
+import { qualifyUrl } from 'util/URLUtils';
+import fetch from 'logic/rest/FetchProvider';
+
+const fetchShowDatanodeMigration = async () => fetch('GET', qualifyUrl('/datanode/configured'));
+
+const useShowDatanodeMigration = () : boolean => {
+  const { data: showDatanodeMigration } = useQuery(
+    ['show_datanode_migration'],
+    fetchShowDatanodeMigration,
+  );
+
+  return showDatanodeMigration === true;
```

Comment:
Sorry, the logic is wrong here. It needs to be the exact opposite. If `/datanode/configured` returns `true`, it should not be shown

Reply:
my bad, I set it to true to test it then forgot it, I fixed it and renamed the variable to avoid any confusion.